src-core/products/processor/image_processor.cpp

```diff
@@ -131,7 +130,7 @@ namespace satdump
                                                                         rgb_image,
                                                                         final_timestamps,
                                                                         *img_products);
-                        ret.img.save_png(product_path + "/rgb_" + name + "_projected.png");
```

Comment:
Thinking a bit more, that sounds like a perfect idea to me. Fully separate the product format. Perhaps keep the current category and system, with an override specific to products and a separate selection box.

As for my previous J2K comment, I think I'll tackle it a bit later today.

Reply:
This could work - want me to add a separate option for "raw product format," then use that one within `ImageProducts` so it affects everything that ends up in a cbor? That should be pretty clean

Reply:
Implemented as of 03ffe59b - open to thoughts!